Adding litellm to relevant resources (#757)
openai/openai-cookbook: related_resources.md

- [Guidance](https://github.com/microsoft/guidance): A handy looking Python library from Microsoft that uses Handlebars templating to interleave generation, prompting, and logical control.
- [Haystack](https://github.com/deepset-ai/haystack): Open-source LLM orchestration framework to build customizable, production-ready LLM applications in Python.
- [LangChain](https://github.com/hwchase17/langchain): A popular Python/JavaScript library for chaining sequences of language model prompts.

> - [LiteLLM](https://github.com/BerriAI/litellm): Easily call Anthropic/Huggingface/etc. using the OpenAI format 

Reviewer: ```suggestion
- [LiteLLM](https://github.com/BerriAI/litellm): Minimal Python library that allows you to call various LLM APIs with a consistent format. 
```

> - [🚅 LiteLLM](https://github.com/BerriAI/litellm): Easily call Anthropic/Huggingface/etc. using the OpenAI format 

Author: is that a requirement? This is a pretty consistent part of our branding, and I'd prefer to keep it.

> - [🚅 LiteLLM](https://github.com/BerriAI/litellm): Easily call Anthropic/Huggingface/etc. using the OpenAI format 

Reviewer: Yes that's a requirement. We don't want to give preferential treatment to any library, and emojis stand out.

> - [🚅 LiteLLM](https://github.com/BerriAI/litellm): Easily call Anthropic/Huggingface/etc. using the OpenAI format 

Author: ok - updated.

> - [LiteLLM](https://github.com/BerriAI/litellm): A Python library to call various LLM APIs (OpenAI/Azure/Huggingface/etc.) using the OpenAI format 

Reviewer: ```suggestion
- [LiteLLM](https://github.com/BerriAI/litellm): A minimal Python library for calling LLM APIs with a consistent format. 
```
